src/compose.ts

```diff
@@ -226,7 +250,7 @@ function processChanges(composer: Compose) {
   const changes = composer.changes;
   composer.changes = Object.create(null);
 
-  if (!('view' in changes) && !('viewModel' in changes) && ('model' in changes)) {
+  if (!('view' in changes) && !('viewModel' in changes) && ('model' in changes) && determineActivationStrategy(composer) !== ActivationStrategy.Replace) {
```

Comment:
it feels to me the whole `!('view' in changes) && !('viewModel' in changes) && ('model' in changes)` should be in determine activation strategy as well, or abstracted to a separate function. But it just feeling, feel free to ignore this 😀

Reply:
Actually that can be done. This stuff can be put in `determineActivationStrategy`, which normalizes the whole thing to a binary decision, activate or instantiate.

Reply:
Refactored the implementation. Hopefully the changes make more sense now.